### DISCUSSION TITLE
Switching to a new Sphinx Documentation Theme

### DISCUSSION THREAD

**Discussion by jeremyleung521** [westpa/westpa#326] [Ideas]
Current Sphinx: https://westpa.readthedocs.io

We are currently using the `sphinx_rtd_theme` for our sphinx documentation. That theme is somewhat slow on getting updated (No Sphinx 7 support yet) and a lot of the scientific community are moving on to the`pydata_sphinx_theme` ([link](https://github.com/pydata/pydata-sphinx-theme)). It's also more compatible with the DIATAXIS Framework we've been using in our wiki. Just wondering what's everyone's thought on it?

The PyData theme and Pandas using the theme
https://pydata-sphinx-theme.readthedocs.io/en/stable/
https://pandas.pydata.org/docs/

**Comment by jpthompson17** [comment]
I've been using the PyData theme to draft reference docs, and I really like the look and feel. I'm all for switching.
